src/main/java/appeng/client/render/effects/LightningFX.java

```diff
@@ -31,14 +31,14 @@
 
 public class LightningFX extends EntityFX
 {
-
 	private static final Random RANDOM_GENERATOR = new Random();
+
 	final int steps = this.getSteps();
 	final double[][] Steps;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename field "Steps" to prevent any misunderstanding/clash with field "steps" defined on line 36. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1845)